README.md

```diff
@@ -153,7 +153,10 @@ When running feature specs, the default behavior is for exceptions to be rescued
 $ RAISE=true rspec
 ```
 
-If you encounter issues running features specs, check the version of chromedriver you have installed.  Version 2.38 is known to work.
+If tests return with `Too many open files`, try
+```sh
+$ ulimit -n 8192
```

Comment:
🎉 
Didn't realize you could do this - kept having to do it on the command line (well.. eventually added to my .zshrc)